src/__tests__/downshift.misc.js

```diff
@@ -7,6 +7,16 @@ import ReactDOM from 'react-dom'
 import {render} from 'react-testing-library'
 import Downshift from '../'
 
+const oldError = console.error
+
+beforeEach(() => {
+  console.error = jest.fn()
+})
+
+afterEach(() => {
+  console.error = oldError
+})
+
```

Comment:
I think this could be easier:

```javascript
beforeEach(() => jest.spyOn(console, 'error').mockImplementation(() => {}))
afterEach(() => console.error.mockRestore())
```

Reply:
done